cypress/integration/leftNav.spec.js

```diff
@@ -86,7 +86,7 @@ describe('Left-nav test', () => {
     cy.contains('button', 'Item of').click()
     // Wait for scroll
     // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(1000)
+    cy.wait(1500)
```

Comment:
Sorry to have to increase this, but I couldn't get it to pass locally with just `1000`. :unamused: